channels/api.py

```diff
@@ -606,17 +626,18 @@ def remove_moderator(self, moderator_name, channel_name):
             # and the double removal case is probably more common.
             pass
 
-    def list_moderators(self, channel_name):
+    def list_moderators(self, channel_name, moderator_name=None):
         """
         Returns a list of moderators for the channel
 
         Args:
             channel_name(str): name of the channel
+            moderator_name(str): username of the user
 
         Returns:
             praw.models.listing.generator.ListingGenerator: a generator representing the contributors in the channel
         """
-        return self.get_channel(channel_name).moderator()
+        return self.get_channel(channel_name).moderator(redditor=moderator_name)
```

Comment:
According to the code comment `redditor` here will restrict the list of moderators to only `moderator_name`. Do we want to do that here?

Reply:
Looking at the rest of the code it looks like we don't use `moderator_name` anywhere. Did you mean to use it in `ModeratorDetailView`?

Can you also update the docstring to say that `moderator_name` will do filtering if specified?

Reply:
I was using this at one point, and this is possibly useful in other places where we're just trying to check for a moderator's existence. I can back this out for now though since I'd prefer to keep the delta down for this PR.